avm/res/aad/domain-service/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.27.1.19265",
-      "templateHash": "11541389602215050557"
+      "version": "0.28.1.47646",
```

Comment:
I'd recommend to only regenerate the readme, without compiling the json file, unless changes are done to the main.bicep template. 
The reason is that this will also trigger the publishing and bump the patch version of the module

Reply:
Small correction @eriqua (a well hidden secret): You can run the `Set-AVMModule` script simply with the `-SkipBuild` flag. Buja!
fyi: @ReneHezser

Reply:
Or that then 😄

Reply:
The PR now looks good to me. @ReneHezser I would however suggest you test it yourself again after the recent changes to make sure the code still works as expected.
Aside from that I'll leave the last word to @eriqua given this comment :)

Reply:
Makes sense. I'll test and ping here when done.

Reply:
Some tests are failing due to WAF rules.
Only the [network/load-balancer](https://github.com/ReneHezser/bicep-registry-modules/actions/runs/9517624389) test fails in the readme. I think the file was not up-to-date, as the outcome of the test seems reasonable.

Reply:
Hey Rene, don't worry about those. This PR is only concerned with the static Pester tests

Reply:
@eriqua ready when you are ;-)